wizeline/remix-project-lab
# Feature/95/archive project

# What does this PR do?
I added the option of archive and unarchived projects. Admins and project members could archive and unarchived projects.
The archive projects will be hidden in the active list of projects. But you will find it on your proposal list.



# How should this be manually tested?

1. Go to the main page of a project
2. Click on the archive button
3. Confirm the action
4. Go to the active projects list, you will not find it
5. Go to the my proposal list projects
6. Look for the archive project and select it
7. Click on the unarchive project
8. Confirm the action
9. Go to the active projects list, you will find it




# What are the relevant tickets?
#95 


# Screenshots
![image](https://user-images.githubusercontent.com/74632242/210286765-e1fb3e6b-8af8-4714-9921-32b22c1a1f89.png)
![image](https://user-images.githubusercontent.com/74632242/210286769-f2c5e247-e1b0-4587-996d-9fa3e31c0970.png)
![image](https://user-images.githubusercontent.com/74632242/210286815-eedbac0d-0c7b-4653-bd52-1a4cdf232f11.png)
![image](https://user-images.githubusercontent.com/74632242/210286793-865fe6e6-b699-4840-bbd5-b8a6fd9b2f32.png)
![image](https://user-images.githubusercontent.com/74632242/210286833-802bb4b7-c62a-44e7-9f7a-033f3b09b654.png)

# Checklist
